New-Framework/Elements/AppMenu.cs

```diff
@@ -1,11 +1,4 @@
 ﻿using TestStack.White.UIItems.Finders;
-using UnitTestProject1.appl;
-using TestStack.White;
-using TestStack.White.UIItems.WPFUIItems;
-using TestStack.White.UIItems.MenuItems;
-using TestStack.White.ScreenObjects.ScreenAttributes;
-using System.Linq;
-using System.Windows.Controls;
 
 namespace UnitTestProject1.Elements
 {
```

Comment:
What a reason of existing this class?
There are no specific methods for a menu

Reply:
updated the class with ClickMenuItem() method